frontend/src/components/commons/DisplaySplunk.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { FaCheck, FaExclamationCircle, FaExclamationTriangle } from "react-icons/fa";
+import {formatTime} from "../../helpers/Formatters";
+import {SplitView} from "../PatternflyComponents/Split/SplitView";
+import CardView from "../PatternflyComponents/Card/CardView";
+import ListView from "../PatternflyComponents/List/ListView";
+import {Text6} from "../PatternflyComponents/Text/Text";
+import { getBuildLink } from './commons';
+
+
+export function DisplaySplunk({benchmarkConfigs}) {
+
+
+
+    const icons = {
+        "failed": <FaExclamationCircle color="red" />,
+        "failure": <FaExclamationCircle color="red" />,
+        "success": <FaCheck color="green" />,
+        "upstream_failed": <FaExclamationTriangle color="yellow" />,
+
+    }
+
+    const {  getSplunkLink, getTimeFormat, status } = getSplunkData(benchmarkConfigs)
+
+    return (
+        <CardView header={"Tasks Ran"}
+                  body={ benchmarkConfigs && <ListView isPlain={true} list_view={
+                                      [<SplitView splitValues={[
+                                                icons[status] || status,
+                                                benchmarkConfigs.benchmark,
+                                                `( ${getTimeFormat} )`,
+                                                getSplunkLink,
+                                                getBuildLink(benchmarkConfigs)]}
+                                      />]
+                                  }
+                  /> || <Text6 value={"No Results"} /> }
+        />
+    )
+
+}
+
+const getSplunkData = (benchmarkConfigs) => {
+    const splunkURL = "https://rhcorporate.splunkcloud.com/en-GB/app/search/";
+    let status = null;
+    let getTimeFormat = null;
+    let getSplunkLink = null;
+    if(benchmarkConfigs){
+        const startDate = new Date(benchmarkConfigs.startDate)
+        const endDate = new Date(benchmarkConfigs.endDate)
+
+        // Adding 10 more minutes for a buffer.
+        endDate.setMinutes(endDate.getMinutes() + 10);
+
+        let getSplunkURL = null;
+        switch(benchmarkConfigs.benchmark) {
```

Comment:
Just a nitpick for a refactor could we move this switch case to a function to remove a lot of lines from this function